.nomad/scripts/prestart.sh

```diff
@@ -0,0 +1,12 @@
+php artisan config:cache --no-interaction --verbose
+php artisan view:cache --no-interaction --verbose
+php artisan event:cache --no-interaction --verbose
+php artisan route:cache --no-interaction --verbose
+php artisan cache:clear --no-interaction --verbose
+php artisan migrate --no-interaction --force --verbose
+mkdir --parents /assets/${NOMAD_JOB_NAME}/
+cp --recursive --verbose public/* /assets/${NOMAD_JOB_NAME}/
+if [ ${PERSIST_RESUMES} = "false" ] && [ ${DB_CONNECTION} = "mysql" ]
+then
+    mysql --execute="update users set resume_date=null"
```

Comment:
I assume you've tested this, but in case you haven't, is a semicolon required here?

Reply:
This has indeed been tested and it works without a semicolon.